### ISSUE TITLE
Quitting right after deleting notifications doesn't delete

### ISSUE THREAD

**Issue by posva** [volta-net/feedback#79]
If you delete some notification in the inbox and then immediately quit while the notification at the bottom right is still there, the notifications are not really deleted. I suppose this is to allow the "Undo" action. I think that some actions like quitting should _commit_ these pending actions before quitting

**Comment by benjamincanac** [comment]
Indeed, the http call happens after the 5s timeout. Do you think it's possible to trigger those http calls when reloading the page for example?

**Comment by posva** [comment]
Maybe in a https://developer.mozilla.org/en-US/docs/Web/API/Window/beforeunload_event they can be sent
